pkg/rhcos/baremetal.go

```diff
@@ -0,0 +1,23 @@
+package rhcos
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// Baremetal fetches the BaseURI where baremetal images can be downloaded from
+func Baremetal() (string, error) {
```

Comment:
There's nothing specific to bare metal about this - I'd just call it rhcos.BaseURI()